src/Python/qsharp-core/qsharp/tests/test_azure.py

```diff
@@ -138,6 +138,10 @@ def _test_workspace_with_providers_after_connection():
     assert isinstance(retrieved_job, AzureJob)
     assert job.id == retrieved_job.id
 
+    # Submit a job with the "jobParams" parameter
+    job2 = qsharp.azure.submit(op, jobParams={"key1": "value1", "key2": "value2"})
```

Comment:
Thanks for adding these tests; looks like the Python side picked up the new parameter correctly, then! Do we need to modify the docstrings accordingly?

Reply:
The docstrings in [azure.py](https://github.com/microsoft/iqsharp/blob/main/src/Python/qsharp-core/qsharp/azure.py) refer to the magic docs, so we're covered.   Added jobParams to examples in the magic docs.